pkg/minikube/node/cache.go

```diff
@@ -99,13 +99,17 @@ func doCacheBinaries(k8sVersion string) error {
 }
 
 // BeginDownloadKicArtifacts downloads the kic image + preload tarball, returns true if preload is available
-func beginDownloadKicArtifacts(g *errgroup.Group) {
+func beginDownloadKicArtifacts(g *errgroup.Group, driver string, cRuntime string) {
 	out.T(out.Pulling, "Pulling base image ...")
-	glog.Info("Beginning downloading kic artifacts")
-	g.Go(func() error {
-		glog.Infof("Downloading %s to local daemon", kic.BaseImage)
-		return image.WriteImageToDaemon(kic.BaseImage)
-	})
+	glog.Infof("Beginning downloading kic artifacts for %s with %s", driver, cRuntime)
+	if driver == "docker" {
```

Comment:
shouldnt this be driver.IsKIC( ?

Reply:
It is driver.IsKIC (in the caller), but this function calls the docker-specific WriteImageToDaemon

Reply:
Let's use oci.Docker not string

Reply:
oci.Docker _is_ a string, but ok. Would be nice to have something generic, in `image` ? Preferrably with a progressbar.